test/unit/tracks/text-tracks.test.js

```diff
@@ -574,7 +574,7 @@ if (Html5.isSupported()) {
     player.addRemoteTextTrack(track, false);
     player.src({src: 'example.mp4', type: 'video/mp4'});
 
-    this.clock.tick(1);
+    this.clock.tick(2);
```

Comment:
Calling `src()` now results in two async functions being queued up in sequence. First, `middleware.setSource` runs async. Then, its callback invokes `this.triggerReady()`, which queues up a flush of the ready queue.

That's why this changed from `1` to `2`.

Reply:
Im just wondering if this is going to break plugin tests, I don't really see an issue other that that.

Reply:
It might.